src/Azure.Functions.Cli/Helpers/SecurityHelpers.cs

```diff
@@ -109,5 +113,28 @@ internal static X509Certificate2 GetOrCreateCertificate(string certPath, string
 
             throw new CliException("Auto cert generation is currently not working on the .NET Core build.");
         }
+
+        internal static X509Certificate2 CreateCertificateOpenSSL()
+        {
+            const string DEFAULT_PASSWORD = "localcert";
+
+            var cert_file_names = Path.Combine(Path.GetTempPath(), Path.GetRandomFileName().Replace(".", String.Empty));
+
+            var openssl_key = new Executable("openssl", $"req -new -x509 -newkey rsa:2048 -nodes -keyout {cert_file_names}localhost.key -out {cert_file_names}localhost.cer -days 365 -subj /CN=localhost");
+            var exitCode = openssl_key.RunAsync(l => ColoredConsole.WriteLine(l)).Result;
```

Comment:
I'd just make all the chain async to avoid calling `.Result`. It's not a big deal, but it just makes the code cleaner.